bootstrap/sql/migrations/native/1.2.0/postgres/schemaChanges.sql

```diff
@@ -131,4 +132,30 @@ SET json = jsonb_set(
 WHERE serviceType = 'DomoPipeline';
 
 -- Query Entity supports service, which requires FQN for name
-ALTER TABLE query_entity RENAME COLUMN nameHash TO fqnHash;
\ No newline at end of file
+ALTER TABLE query_entity RENAME COLUMN nameHash TO fqnHash;
+
+CREATE TABLE IF NOT EXISTS persona_entity (
+  id VARCHAR(36) GENERATED ALWAYS AS (json ->> 'id') STORED NOT NULL,
+  name VARCHAR(256) GENERATED ALWAYS AS (json ->> 'name') STORED NOT NULL,
+  nameHash VARCHAR(256) NOT NULL,
+  json JSONB NOT NULL,
+  updatedAt BIGINT GENERATED ALWAYS AS ((json ->> 'updatedAt')::bigint) STORED NOT NULL,
+  updatedBy VARCHAR(256) GENERATED ALWAYS AS (json ->> 'updatedBy') STORED NOT NULL,
+  PRIMARY KEY (id),
+  UNIQUE (nameHash)
+);
+CREATE INDEX persona_name_index ON persona_entity USING btree (name);
+
+
+CREATE TABLE IF NOT EXISTS doc_store (
```

Comment:
entityType will take care of this so we are not storing the JSON schema with it.
For Pages and KnowledgePanels this is ok. If we need it will address in follow-up PR